bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkMutationWrapper.java

```diff
@@ -60,19 +59,19 @@ public boolean isFlushed() {
 
   /** {@inheritDoc} */
   @Override
-  public ListenableFuture<Void> add(RowMutation rowMutation) {
-    return Futures.transform(delegate.add(rowMutation.toBulkProto(requestContext).getEntries(0)),
+  public ApiFuture<Void> add(RowMutation rowMutation) {
+    return ApiFutureUtil.transformAndAdapt(delegate.add(rowMutation.toBulkProto(requestContext).getEntries(0)),
```

Comment:
Will BulkMutation change to return an ApiFuture as well?

Reply:
The `BulkMutation` delegate has to be backwards compatible, so it will not return `ApiFuture`s.

Reply:
I wasn't aware of it, I just thought all delegation should migrate to `ApiFuture`.

If we continue to use `ListenableFuture` for both operation i.e. BulkMutation#add & BulkMutation#readModifyWrite 
then Can I close this PR? As origin of this PR is based on migrating BulkMutation.

Reply:
@sduskis, Saw your fresh comments, I would change return types to `Future`.

Reply:
Sorry... it's not the return type that needs to change.  `ApiFuture` and `ListeneableFuture` both extend from `Future`.  We should use `Future` in places that don't care about the specific `Future` Extension.